src/rust/engine/src/nodes.rs

```diff
@@ -800,7 +800,7 @@ impl DownloadedFile {
           if new_size > self.size_limit {
             Err(std::io::Error::new(
               std::io::ErrorKind::InvalidData,
-              "Downloaded file was larger than expected digest",
+              format!("Downloaded file size ({}) was larger than expected digest ({}).", new_size, self.size_limit)
```

Comment:
this works.
<img width="980" alt="Screen Shot 2020-10-23 at 2 04 21 PM" src="https://user-images.githubusercontent.com/1268088/97054489-b9d51200-1539-11eb-87fa-e893141527cc.png">

However, I kind of guessed the rust syntax here based on other places (in this file)...

Reply:
The syntax is fine but the message is misleading. You only know the file was limit + fraction of next chunk bigger. You can't say how big the file actually would be from the data here. Perhaps reword to just note the expected size and the fact that size was exceeded.